src/mvt_getter.py

```diff
@@ -199,34 +219,21 @@ def get_vectors(minzoom, maxzoom, x, y, style, vec):
             """create or replace function public.basemap_z%s(x integer, y integer)
         returns bytea
         as $$
-        declare
-            area_mvt bytea;
-            line_mvt bytea;
-            node_mvt bytea;
-            result bytea;
         begin
-            select into area_mvt
-                ST_AsMVT(tile, 'area', 4096, 'way')
-                from (%s) as tile;
-
-            select into line_mvt
-                ST_AsMVT(tile, 'line', 4096, 'way')
-                from (%s) as tile;
-
-            select into node_mvt
-                ST_AsMVT(tile, 'node', 4096, 'way')
-                from (%s) as tile;
-
-            return area_mvt || line_mvt || node_mvt;
+            return (
```

Comment:
if language is sql, just return. on the second thought, are st_asmvt here going to always return not null? I'd put a coalesce(,'') here just to be safe.